src/app/components/Heading/index.tsx

```diff
@@ -0,0 +1,54 @@
+/** @jsx jsx */
+
+import React, { FC } from 'react';
+import { jsx } from '@emotion/react';
+
+import { GelFontSize, FontVariant } from '../../interfaces';
+import Text from '../Text';
+
+interface Props {
+  className?: string;
+  children: React.ReactNode;
```

Comment:
as with other newly developed components like the `Subhead`, may want to use `PropsWithChildren` instead here